lib/manager/mix/extract.js

```diff
@@ -0,0 +1,68 @@
+const { isValid } = require('../../versioning/hex');
+
+module.exports = {
+  extractPackageFile,
+};
+
+function extractPackageFile(content) {
+  logger.trace('mix.extractPackageFile()');
+  const deps = [];
+  const contentArr = content.split('\n');
+  const depSectionRegExp = /defp\s+deps.*do/g;
+  const depMatchRegExp = /{:(\w+),\s*([^:"]+)?:?\s*"([^"]+)",?\s*(organization: "(.*)")?.*}/gm;
+
+  for (let lineNumber = 0; lineNumber < contentArr.length; lineNumber += 1) {
+    if (contentArr[lineNumber].match(depSectionRegExp)) {
+      logger.trace(`Matched dep section on line ${lineNumber}`);
+      let depBuffer = '';
+      do {
+        depBuffer += contentArr[lineNumber] + '\n';
+        lineNumber += 1;
+      } while (!contentArr[lineNumber].includes('end'));
+      let depMatch;
+      do {
+        depMatch = depMatchRegExp.exec(depBuffer);
+        if (depMatch) {
+          const depName = depMatch[1];
+          const datasource = depMatch[2];
+          const currentValue = depMatch[3];
+          const organization = depMatch[5];
+
+          const dep = {
+            depName,
+            currentValue,
+          };
+
+          if (datasource) {
+            dep.datasource = datasource;
+            dep.depType = datasource;
+            dep.lookupName = currentValue;
+            dep.currentValue = null;
+          } else {
+            dep.datasource = 'hex';
+            dep.depType = 'hex';
+            dep.currentValue = currentValue;
+          }
+
+          if (organization) {
+            dep.organization = organization;
```

Comment:
How is this field used?

Reply:
It is used to acces private hex repos

Reply:
How exactly is that done? We don't use an "organization" field in other managers. And it won't be passed to the hex datasource.

Reply:
Organization is a hex term. Users access private repos within a organization. I can resolve this in separate pr for hex data source.